### DISCUSSION TITLE
xmake packages and template library

### DISCUSSION THREAD

**Discussion by explocion** [xmake-io/xmake#2687] [Q&A] [answered]
I'm wondering is there a `set_kind` kind corresponding to template libraries for C++. Template libraries usually does not have any binary or lib files to install, but it need to have its source code installed into `/usr/include` in the form of `.h` header files.

**Comment by waruqi** [comment, marked as answer]
set_kind("headeronly")

**Reply by explocion** [reply]
Thanks!
